crates/doc/src/inference.rs

```diff
@@ -107,6 +107,8 @@ pub enum Reduction {
 
     // Multiple concrete strategies may apply at the location.
     Multiple,
+
+    JsonSchemaMerge,
```

Comment:
nit: order / sort with the block of others. This isn't special, in the way that `Unset` and `Multiple` are.